HolubetsYustyna/HW7/7-3.py

```diff
@@ -0,0 +1,19 @@
+# Write a function that calculates the number of characters included in a given string
+
+def charNumber_1(str) -> dict:
+    """returns the frequencies of characters in a string as dict {'char': freq} )"""
+    return {key: str.count(key) for key in tuple(str)}
```

Comment:
good job
I recommended to use method set()
return {key: str.count(key) for key in set(str)}